setup.py

```diff
@@ -48,7 +49,7 @@
     "pytest-runner>=2.7",
 ]
 
-install_requires = ["click>=7,<8", "requests==2.22.0", "pycurl>=7"]
+install_requires = ["click>=7,<8", "requests==2.22.0"]
```

Comment:
Any reason why we require hard-pinning or `requests` to 2.22.0?  (Ditto for `click`?) If no particular reason, it would be good to release these constraints in order to support older/newer versions as well, so that `cernopendata-client` would be installable on a variety of systems.

If we encounter some limits due to Python versions, such as 2.7, there is always a technique like:

```console
$ git grep python_version setup.py
setup.py:44:    'sqlalchemy-utils>=0.35.0 ; python_version>="3"',
setup.py:45:    'sqlalchemy-utils<=0.36.3 ; python_version=="2.7"',
```

Reply:
I will check it and update the PR.

Reply:
... otherwise the PR works very nicely :rocket:

Reply:
@tiborsimko  I just checked, the latest version of click is `7.1.2` and request is `2.24.0`. I can change the requests to `>=2,<3`, we don't need any change in the click.

Reply:
Perhaps even keep the upper boundaries open? So that we declare only the lowest needs...

Reply:
okay, removed the upper boundaries 👍